### ISSUE TITLE
updatePort.sh script

### ISSUE THREAD

**Issue by rcastberg** [haugene/docker-transmission-openvpn#198]
Hi,

I am having some issues with the updatePort.sh script. Seems that i have inet and inet6, and the script doens't differentiate between them. Changing the  line :
local_vpn_ip=$(ip addr show tun0 | grep inet | awk '{ print $2 }')
to 
local_vpn_ip=$(ip addr show tun0 | grep "inet " | awk '{ print $2 }')
solved the problem

Example output from ip addr show tun0:
```
2: tun0: <POINTOPOINT,MULTICAST,NOARP,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UNKNOWN group default qlen 100
    link/none 
    inet 10.01.01.01 peer 10.01.01.01/32 scope global tun0
       valid_lft forever preferred_lft forever
    inet6 aaaa::aaaa:aaaa:aaaa:aaaa/64 scope link 
       valid_lft forever preferred_lft forever

```

**Comment by haugene** [comment]
Good catch, and your proposal for a solution should do the trick. Just merged #202 (a fix for #201) which is also about the updatePort script. Will test those changes when I have the time. Then I can try this fix as well.

**Comment by haugene** [comment]
Hmm, had a closer look at this now. It seems the line you're referring to was removed in april last year. So your image is quite old. The issue should be resolved by pulling the image again and recreating your container.

Feel free to re-open this issue if it doesn't solve your problem.
